src/client.js

```diff
@@ -150,6 +178,61 @@ class PureCloudWebrtcSdk extends WildEmitter {
       }
     });
   }
+
+  _log (level, message, details) {
+    // immediately log it locally
+    this.logger[level](message, details);
+
+    // ex: if level is debug and config is warn, then debug is less than warn, don't push
+    if (LOG_LEVELS.indexOf(level) < LOG_LEVELS.indexOf(this._logLevel)) {
+      return;
+    }
+
+    if (this._optOutOfTelemetry) {
```

Comment:
How do you actually opt out here? Not seeing a way to supply via options (meant to be kept private for internal use?)

Reply:
Yeah, you can just set this directly on the sdk instance. Meant to be a little hidden.